### ISSUE TITLE
Crash at loading - net.minecraftforge.fml.common.LoaderExceptionModCrash

### ISSUE THREAD

**Issue by mikolajww** [danikk14g/Craftologia#1]
System info: 
Windows 7 64bit
i5-4460, GTX 1060, 8GB RAM

Version:
Craftologia-1.0.b-hotfix1

What happened:
Crashed during launch with MultiMC 0.6.2-develop-1245
Last log in console:
```
		* zerocore|API|multiblock (1.10.2-0.0.2) from zerocore-1.12-0.1.2.2.jar
		* zerocore|API|multiblock|rectangular (1.10.2-0.0.2) from zerocore-1.12-0.1.2.2.jar
		* zerocore|API|multiblock|tier (1.10.2-0.0.2) from zerocore-1.12-0.1.2.2.jar
		* zerocore|API|multiblock|validation (1.10.2-0.0.2) from zerocore-1.12-0.1.2.2.jar
	[DummyCore]: 'Special case ASM modification mods: ''Note, that these mods might not be involved in the crash in ANY WAY!''DummyCore just prints some known mods for a lot of ASM modifications''dummycore''EnderCore''CodeChickenLib'
[12:04:35] [main/INFO] [STDOUT]: [net.minecraft.init.Bootstrap:func_179870_a:553]: #@!@# Game crashed! Crash report saved to: #@!@# C:\Users\Mikolaj\Desktop\Things n' stuff\MultiMC\instances\Craftologia-1.0.b\minecraft\crash-reports\crash-2018-08-17_12.04.35-client.txt
Process exited with code -1.
```
Attached crash log
[crash-2018-08-17_12.04.35-client.txt](https://github.com/danikk14g/Craftologia/files/2296813/crash-2018-08-17_12.04.35-client.txt)

**Comment by Pandemoneus** [comment]
If you update both ExtendedDays (ExtendedDays-1.12.2-0.2.6-22) and SilentLib (SilentLib-1.12.2-2.3.18-140), the crash will be resolved.

[edit]
And boom, when you try creating and loading a world, you get another ExtendedDays crash with the updated version:
https://hastebin.com/catucijepo.cs

**Comment by dancc7t** [comment]
i am using in my testing/playing world versions SilentLib-1.12.2-2.3.16-136 and ExtendedDays-1.12-0.2.5-22, i changed to these in version 1.0.c, which i hope solves once for all these problems with extended days, so people dont have to disable it!

**Comment by mikolajww** [comment]
Ok, the loading issue is resolved in the version 1.0.c.
